profile/models/run_time_standard/run_time_standard.cpp

```diff
@@ -9,8 +9,12 @@
 int main(int argc, char* argv[]) {
 
 #ifdef MPIx
-  MPI_Init(&argc, &argv);
+  int provided;
+  MPI_Init_thread(&argc, &argv, MPI_THREAD_MULTIPLE, &provided);
```

Comment:
Actually I've been thinking about this today and MPI_THREAD_SERIALIZED is enough
All our calls to MPI routines are done by the main thread, except for the MPI_ABORT call in the case of an error (so MPI_THREAD_FUNNELED is almost enough)

Reply:
On at least one cluster I used most libraries installed did not provide MPI_THREAD_MULTIPLE, but lower levels of thread supports were ok